definitions/ext-snowflake_account/golden_metrics.yml

```diff
@@ -0,0 +1,76 @@
+warehouseCount:
+  title: "How many virtual warehouses in your Snowflake account"
+  query:
+    select: uniqueCount(NAME)
+    from: SnowflakeAccount
+    where: "metric_type = 'snowflake.billing'"
+    eventId: entity.guid
+  displayAsValue: false
```

Comment:
Yep, you are right, this is the default so it can be removed (unless they want it explicit). The `eventId` is also the default one so it could be removed if they want.

Reply:
should I remove them?

Reply:
You can leave them if it's clearer for you, but otherwise, you can go with the defaults.